R/utility_spliceHelper_functions.R

```diff
@@ -1,598 +1,572 @@
-# Note: several of the functions in this file are adopted from 
-# the GenomicAlignments package (Author: Hervé Pagès, Valerie Obenchain,
-# Martin Morgan)
-# License Artistic-2.0
+# Note: several of the functions in this file are adopted from the GenomicAlignments package (Author: Hervé Pagès, Valerie Obenchain, Martin Morgan)
+# License	Artistic-2.0
 # https://doi.org/doi:10.18129/B9.bioc.GenomicAlignments
 
-## examples for test purposes
```

Comment:
Can you document the test data somewhere using these descriptions? Where would be the best place to document the data?

Reply:
I have added the document of the test data back on top of the compareTranscripts() function.